src/statistics.sh

```diff
@@ -17,83 +17,21 @@ declare -gA shared_data=(['deploy']='' ['build']='' ['list']='' ['uninstall']=''
 
 function statistics()
 {
-  local target="$1"
-  local day
-  local week
-  local month
-  local year
-
-  if [[ "$1" =~ -h|--help ]]; then
-    statistics_help "$1"
-    exit 0
-  fi
-
-  shift 1 # Remove the first option, i.e., --day, --week, or --year
-
   if [[ "${configurations[disable_statistics_data_track]}" == 'yes' ]]; then
     say 'You have disable_statistics_data_track marked as "yes"'
     say 'If you want to see the statistics, change this option to "no"'
     return
   fi
 
-  # Default to day
-  if [[ -z "$target" ]]; then
-    target='--day'
+  if [[ -n "${options_values['DAY']}" ]]; then
+    day_statistics "${options_values['DAY']}"
+  elif [[ -n "${options_values['WEEK']}" ]]; then
+    week_statistics "${options_values['WEEK']}"
+  elif [[ -n "${options_values['MONTH']}" ]]; then
+    month_statistics "${options_values['MONTH']}"
+  elif [[ -n "${options_values['YEAR']}" ]]; then
+    year_statistics "${options_values['YEAR']}"
```

Comment:
Currently, `statistics` is the entry point in the `kw` file for the statistic feature. After your refactor, I expect that this code manipulation will abode in the report file.